js/ui/tabs.js

```diff
@@ -425,6 +425,12 @@ const Tabs = CollectionWidget.inherit({
         this.callBase();
     },
 
+    _updateScrolling($target) {
```

Comment:
Yes, it's a good name. I fixed it in this commit [402689d46ecda102c8bc0f40b9fcc6495520eb83].
Yes, you are right: not only disabled items should be visible on focus. Another items become visible too, because the handler is called for them when the selectedItem option is changed in the collection widget. It is difficult logic, we don't have to fix this.

Reply:
Let's discuss it in voice

Reply:
I suggest u just always call `scrollToItem` in option changed

Reply:
Yes, was fixed in [c59a0376565c8f3ab5d954d90c8f08500b96ac08]. Thanks!